### ISSUE TITLE
FE - Auth Profile - Notification Preferences - Update content on FE

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#39341]
## Background

As a FE engineer, I want to update the content on the Notification Preferences page to match the mock-ups so that they are consistent and contain the correct verbiage.

## Tasks
- [ ] Update the content on the Notification Preferences page to match the latest mock-ups
- [ ] Deploy to Staging
- [ ] Update PM & PO when they are ready for review

## Acceptance Criteria
- [ ] Update the content on the Notification Preferences page to match the latest mock-ups
- [ ] Deploy to Staging
- [ ] Update PM & PO when they are ready for review

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by SKasimow** [comment]
Note: awaiting final content changes

**Comment by SKasimow** [comment]
@cgednalske - do you know if we received the final content changes for the notifications page? If so, can you point me to the newest mock?

**Comment by SKasimow** [comment]
Hi @cgednalske - going thru the backlog and came across this ticket. Do you have the updated content that replaces what's on the current notification preference page?

**Comment by cgednalske** [comment]
Hi @SKasimow! I'm a little thrown off by this timeline because I don't think we were actually waiting on content when the first comment here was added. We've had the content set for quite a while. 

The latest Payment section content is what is in the [Notification Preferences Sketch files](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c). I just checked and this was also already updated in staging (unsure about prod). Let me know if I'm missing something, but I don't think we're waiting for anything else for this - at least not from the Payment info work I did. 🙂
